aibolit/utils/ast.py

```diff
@@ -0,0 +1,15 @@
+import javalang
+
+
+class Ast:
+    """
+    Returns the AST for some java file
+    """
+
+    def __init__(self, filename: str):
+        self._filename = filename
+
+    def value(self) -> javalang.ast.Node:
```

Comment:
@paulodamaso I think we need unit test for this class.
For instance we can also test which kind of files would work successfully.